specification.md

```diff
@@ -306,7 +306,7 @@ implementation of this API. We list two:
 1. Treat `JobExecutor` as an abstract base class and have concrete
 subclasses of `JobExecutor` implement the specific mechanisms. The
 subclasses can then be instantiated either directly, using a factory
-pattern, or any other reasonable mechanism. For example:
+pattern or any other reasonable mechanism. For example:
```

Comment:
It may not be clear from the context, but thought I should mention that the the sentence is of the form "&lt;item1&gt;, &lt;item2&gt; or &lt;item3&gt;".